app/main.py

```diff
@@ -1 +1,42 @@
-#  write your code here
+class Animal:
+
+    def __init__(self, name: str, appetite: int, is_hungry: bool = True):
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self):
```

Comment:
Everything look really good, but add type annotations to all your functions.
If function returns nothing then it is -> None